Configuration/Skimming/python/Skims_PDWG_cff.py

```diff
@@ -282,6 +282,18 @@
     dataTier = cms.untracked.string('AOD')
     )
 
+from Configuration.Skimming.PDWG_EXOCSCCluster_cff import *
+EXOCSCClusterPath = cms.Path(EXOCSCClusterSkimSequence)
+SKIMStreamEXOCSCCluster = cms.FilteredStream(
+    responsible = 'PDWG',
+    name = 'EXOCSCCluster',
+    paths = (EXOCSCClusterPath),
+    content = skimRawAODContent.outputCommands+['keep *_csc2DRecHits_*_*','keep *_dt1DRecHits_*_*'],
+    selectEvents = cms.untracked.PSet(),
+    dataTier = cms.untracked.string('AOD')
```

Comment:
I think I'm just taking the value of `skimRawAODContent` output command and adding the extra collections that I need here.
What should I put for the data tier here then?

Reply:
I followed the convention in this PR (similar skim for DTClusters):
https://github.com/cms-sw/cmssw/pull/37509/files

Reply:
@kskovpen so it this really supposed to be `AOD` after all?

Reply:
I wonder if we should switch to RAW-RECO which should better reflect the contents. @sam7k9621 what do you think? Do we have any other skims following this scheme?

Reply:
I think it can be modified as "RAW-AOD".

Reply:
(hmm it's not supposed to be my business, but since I'm following the PR...) isnt `USER` only meant for private productions from the users? I saw the other skims mostly use `RAW-RECO`, why shouldnt this use that too?

Reply:
In my understanding, different “types” only affect how it will be categorized in the DAS.
It uses the same content with EXODisplacedJet, so I think it can follow the type.

Reply:
IMO this should be `USER`, it's `AOD` with customized event content.

Reply:
@kakwok please modify it to `USER` then